pkg/graveler/graveler.go

```diff
@@ -399,7 +399,8 @@ type Branch struct {
 	CommitID     CommitID
 	StagingToken StagingToken
 	// SealedTokens - Staging tokens are appended to the front, this allows building the diff iterator easily
-	SealedTokens []StagingToken
+	SealedTokens    []StagingToken
+	BaseMetaRangeID MetaRangeID
```

Comment:
Add comment explaining what this is.

Reply:
I would also easily select `BaseMetaRangeID` instead of fetch the commit and use its `MetaRangeID`, can we choose something more indicative?